src/main/java/bowling/state/Miss.java

```diff
@@ -16,12 +17,26 @@ public Miss(final Pin first, final Pin second) {
   }
 
   @Override
-  public String score() {
-    return first.totalDownPin().toString() + SEPARATOR + second.totalDownPin().toString();
+  public String scoreMessage() {
+    return first.toString() + SEPARATOR + second.toString();
   }
 
   @Override
   public Pin totalPin() {
     return second.totalDownPin(first);
   }
+
+  @Override
+  public Score score() {
+    return Score.miss(totalPin().pinCount());
+  }
+
+  @Override
+  public Score calculateScore(Score score) {
+    score = first.sumScore(score);
+    if(score.isFinishBallCount()){
```

Comment:
```suggestion
    if (score.isFinishBallCount()) {
```

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
수정하여 반영하겠습니다. 제 커밋 이력을 보시면 아시겠지만 나름대로 개발이 다 끝나서 반영을 한 뒤 전체적으로  Ctrl + Alt + L.을 돌려서 파일 컨밴션을 수행해서 커밋을 하는데요.. 자꾸 한 두건식 누락이 생기네요.